File: wear/src/main/java/com/matejdro/wearvibrationcenter/mutepicker/MuteModeActivity.java
package com.matejdro.wearvibrationcenter.mutepicker;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.google.android.gms.common.api.GoogleApiClient;
import com.google.android.gms.common.api.ResultCallback;
import com.google.android.gms.wearable.CapabilityApi;
import com.matejdro.wearutils.companionnotice.WearCompanionWatchActivity;
import com.matejdro.wearvibrationcenter.R;
import com.matejdro.wearvibrationcenter.common.CommPaths;

public class MuteModeActivity extends WearCompanionWatchActivity implements GoogleApiClient.ConnectionCallbacks, ResultCallback<CapabilityApi.GetCapabilityResult> {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_mute_mode);
    }

    public void openTimedMute(View view) {
        startActivity(new Intent(this, TimedMuteActivity.class));
        finish();
    }

    public void openAppMute(View view) {
        startActivity(new Intent(this, AppMuteActivity.class));
        finish();
    }

    @Override
    public String getPhoneAppPresenceCapability() {
        return CommPaths.PHONE_APP_CAPABILITY;
    }
}

File: wear/src/main/java/com/matejdro/wearvibrationcenter/mutepicker/MuteModeActivity.kt
package com.matejdro.wearvibrationcenter.mutepicker

import android.content.Intent
import android.os.Bundle
import android.view.View
import com.matejdro.wearutils.companionnotice.WearCompanionWatchActivity
import com.matejdro.wearvibrationcenter.R
import com.matejdro.wearvibrationcenter.common.CommPaths
import com.matejdro.wearvibrationcenter.mutepicker.AppMuteActivity

class MuteModeActivity : WearCompanionWatchActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_mute_mode)
    }

    fun openTimedMute(view: View?) {
        startActivity(Intent(this, TimedMuteActivity::class.java))
        finish()
    }

    fun openAppMute(view: View?) {
        startActivity(Intent(this, AppMuteActivity::class.java))
        finish()
    }

    override fun getPhoneAppPresenceCapability(): String {
        return CommPaths.PHONE_APP_CAPABILITY
    }
}
